src/components/Components/DrawerMenu.tsx

```diff
@@ -49,16 +48,22 @@ function DrawerMenu(props :DrawerMenuProps) {
   const { t } = useTranslation();
   const theme = useTheme();
 
+  const isExtraSmallScreen = useMediaQuery((theme:any) => theme.breakpoints.only('xs'));
+  const isSmallScreen = useMediaQuery((theme:any) => theme.breakpoints.only('sm'));
+  const isMediumScreen = useMediaQuery((theme:any) => theme.breakpoints.only('md'));
+
+  const heightBottomBar = isExtraSmallScreen ? 14 : isSmallScreen ? 14 : isMediumScreen ? 7 : 7;
+
```

Comment:
Do we have to hardcode this? The text s clipped on my laptop even at full screen.

Reply:
I am not sure what other solution could we apply here, I will investigate it.

Reply:
Do you get the scroll bar even in full screen? if so we can tweak the number, if not it could be specific to my machine/configuration. Thanks for the investigation